app/src/main/java/com/example/tungcon/demo/data/source/RemoteDataSource.java

```diff
@@ -0,0 +1,7 @@
+package com.example.tungcon.demo.data.source;
+
+import com.example.tungcon.demo.data.Movie;
+
+public interface RemoteDataSource {
+    void getData(CallBack<Movie> callBack);
```

Comment:
em nên đặt tên là getMovies rõ ràng hơn 
```suggestion
    void getMovies(CallBack<Movie> callBack);
```